processing/src/main/java/io/druid/segment/IndexMaker.java

```diff
@@ -610,6 +611,19 @@ public String apply(@Nullable String input)
     progress.stopSection(section);
   }
 
+  static boolean isNullColumn(Iterable<String> dimValues)
```

Comment:
can we have this in another file? My feeling is that we will remove IndexMaker at some piont in the future